XenAdmin/Wizards/NewPolicyWizard/NewPolicySnapshotFrequencyPage.cs

```diff
@@ -45,11 +45,12 @@ namespace XenAdmin.Wizards.NewPolicyWizard
 {
     public partial class NewPolicySnapshotFrequencyPage : XenTabPage, IEditPage
     {
-        public NewPolicySnapshotFrequencyPage()
+        public NewPolicySnapshotFrequencyPage(bool isVMSS)

```

Comment:
This file still needs to have the duplicate code removed.
_copy and _copyVMSS should be declared as IVMGroup. Then all the other instances of if(isVMSS) can be removed. (IVMGroup.frequency will have to return a new enum rather than vmss_frequency or vmpp_backup_frequency).